### DISCUSSION TITLE
Frame Number

### DISCUSSION THREAD

**Discussion by Johntron1219** [bonsai-rx/bonsai#994] [Q&A] [answered]
Hi All, 

Still getting used to bonsai and really appreciate the support (thanks Bruno for the response yesterday!). 

Another question- if I wanted to output the frame number from a file-capture video to tag instances in a CSV file, how would I do that? I've looked everywhere and cannot seem to find anything- is this possible? 

Thanks, 
John

**Comment by bruno-f-cruz** [comment, marked as answer]
Hi,
You can use `ElementIndex` or `Int(1)` + `Accumulate`
![image](https://user-images.githubusercontent.com/7049351/182913669-44682db1-a258-44c7-9819-d3dbad9f7175.png)
cheers,
Bruno

**Reply by Johntron1219** [reply]
Thanks again Bruno!
